### ISSUE TITLE
Provide audible notifications when new events arrive

### ISSUE THREAD

**Issue by justdave** [justdave/nwsweatheralertswidget#4]
What good are weather alerts if you can only see them?  Important stuff like watches and warnings should have an audible notification.  In the case of warnings it ought to be persistent/constant until it's acknowledged.

**Comment by yooperbsn** [comment]
I agree, this would be a nice feature. Got nailed by a pop-up, no warning tornado in the thumb last year.
